### PR TITLE
update metrics docs

### PR DESCRIPTION
**Pull request by vivianyentran**
- Copyedits to Metrics & Metrics Explorer product pages
- Update Metrics product page to mention observability & add timer as a metric type
- Update Metrics Explorer to replace images with arcades

Preview pages:
- https://sentry-docs-git-update-metrics.sentry.dev/product/metrics/
- https://sentry-docs-git-update-metrics.sentry.dev/product/metrics/metrics-explorer-page/

### CODE REVIEW

File: docs/product/metrics/metrics-explorer-page.mdx [hunk 1]
```diff
@@ -10,27 +10,53 @@ Once you've started sending metrics to Sentry, you can explore them in the produ
...
+This will display query results in the same chart, but you can also choose to display them in separate charts. In order to do that click "One chart per query" toggle.
 
-![Equations](./img/equations.png)
+Every query gets its own one-letter identifier that is also shown in the summary table of the chart and allows you to easily distinguish between results of different queries.
+
+Multiple queries can be combined into equations that allow you to perform arithmetic operations on the results of your queries. This can be useful when you want to compare two metrics or calculate a ratio between them. Click "Add Equation" to create a new equation. You can reference queries using one-letter indicators in the input field that appears. For example, if you have two queries a and b, you can create an equation that calculates the ratio between them by typing `a / b`. You can also use the `+`, `-`, and `*` operators.
```
[the remark this answers is not in the document]
Comment (vivianyentran) [reply docs/product/metrics/metrics-explorer-page.mdx:59]: Adjusted the wording in your suggestion to reflect this better

**Comment by vivianyentran** [comment]
Thanks for the suggestions @iambriccardo Appreciate your review :)

**Review by iambriccardo — APPROVED** [review]
Thanks so much for your work!

File: docs/product/metrics/metrics-explorer-page.mdx [hunk 2]
```diff
@@ -10,27 +10,53 @@ Once you've started sending metrics to Sentry, you can explore them in the produ
 
 ## Querying & Visualization
 
-![Metrics Query](./img/metrics-explorer-query.png)
-
-You can use the dropdowns at the top of the page as a query editor for your metrics. Select the metric you want to plot from a list of available metrics in your project, the aggregate operation you'd like to perform (for example avg, min, max, percentiles, and so on), and, optionally, any tags you'd like to group by. In the image above, the p95 values of the metric, `gibpotato.potatoes.event_processing_time`, is queried and shown, grouped by `event_type`.
-
-Optionally, you can also use the search bar to filter for specific tag values. In addition to custom metrics that are defined by your team, you can also explore any performance metrics that Sentry offers out-of-the-box.
+<div style={{"height":"0px","paddingBottom":"calc(56.8359% + 41px)","position":"relative","width":"100%"}}>
```
Comment (ale-cota) [inline docs/product/metrics/metrics-explorer-page.mdx:13]: Small nit, but I noticed that the arcade hint points to the p99 value in the aggregation drop-down, but the instruction says "let's look at the p95 values" and similarly to "release" when it's talking about "event type"

**Review by ale-cota — APPROVED** [review]

File: docs/product/metrics/metrics-explorer-page.mdx (hunk 2 above)
Comment (ale-cota) [reply docs/product/metrics/metrics-explorer-page.mdx:13]: I see now that this is a general issue with arcades, but only in Safari, not in Chrome. The pointer is somehow shifted 🙈

Comment (vivianyentran) [reply docs/product/metrics/metrics-explorer-page.mdx:13]: Oh that's interesting! I had no idea the pointer shifts in other browsers. Thanks for the thorough review!

**Pull request merged**